pkg/eve/installer/grub_installer.cfg

```diff
@@ -63,3 +63,11 @@ if [ "$rootlabel" = "EVEISO" ]; then
       set_global rootfs_root "LABEL=$rootlabel rootimg=/rootfs_installer.img"
    fi
 fi
+
+# include config.img grub, if it exists
+if [ -f "($install_part)/config.img" ]; then
```

Comment:
this is not true for ISO installer... in the ISO image, the config.img stays inside the rootfs_installer.img, not in the rootfs of the image....